app/src/main/java/au/com/shiftyjelly/pocketcasts/ui/MainActivity.kt

```diff
@@ -219,7 +220,10 @@ class MainActivity :
             activity = this
         )
 
-        setupPlayerViews()
+        val showMiniPlayerImmediately = savedInstanceState?.getBoolean(SAVEDSTATE_MINIPLAYER_SHOWN, false) ?: false
+        binding.playerBottomSheet.isVisible = showMiniPlayerImmediately
```

Comment:
Am I understanding correctly that we're updating the visibility here even though it will get updated later in order to avoid (or at least reduce) the miniplayer "flickering" when changing themes? If so, it might be worth adding a comment here briefly explaining that.